imagetest/test_suites/windows/driver_package_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build cit
+// +build cit
+
+package windows
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+func getDriverList(remove bool) []string {
+	drivers := []string{
+		"google-compute-engine-driver-netkvm",
```

Comment:
This is the Network VirtIO driver, so I believe you need to treat it like the gvnic driver were it doesn't get uninstalled.

Reply:
Done.